requirements/pip.txt

```diff
@@ -31,7 +31,7 @@ pyyaml==5.4.1  # pyup: ignore
 Pygments==2.10.0
 
 # Basic tools
-redis==4.0.2
+redis
```

Comment:
You should pin to a specific version here and ignore it from pyup. Besides, add a small comment explaining why it is pinned to an old version.